dpx/src/dpx_cmap.rs

```diff
@@ -1006,70 +980,54 @@ unsafe fn check_range(
     }
     0i32
 }
-static mut __cache: *mut CMap_cache = std::ptr::null_mut();
+static mut __cache: Vec<Box<CMap>> = Vec::new();
```

Comment:
Why not just `Vec<CMap>`?

Reply:
I'm currently boxing the cache entries to be able to get a stable `*mut` pointer even after Vec's backing storage is reallocated for resizing. These `Box`es should probably be replaced with `Rc`s soon.